Added Studio docs (#2455)
iterative/dvc.org: content/docs/studio/experiments.md

> # Run experiments

Reviewer: ```suggestion
# Run Experiments
```

You can also run your ML experiments directly from DVC Studio. Your regular
CI/CD set-up is used to run the experiments. For instance, Github Actions can be

> defined for your Github repositories, to run the ML experiment on any new
> commit.

Reviewer 2: ```suggestion
used to run ML experiments on each new commit.
```

DVC Studio provides you a simple user interface to specify all the changes that
you want to make to your input datasets and hyperparameters. Once you have made
all the necessary changes, you can commit the changes to your Git repository
directly from Studio and invoke your CI/CD pipeline.

To run experiments from DVC Studio, select the commit that you want to use and

> click on the Run button. An input form will be generated for you to enter all
> the input files and parameters that you want to use in your experiment.

Reviewer 2: ```suggestion
click on the `Run` button. An form will be generated for you to fill in all the
input files and parameters needed for your experiment.
```

> <img src="/img/studio/cml.png" alt="drawing" width="300"/>

Reviewer 2: why not markdown?

Author: Wanted to specify width, coz it's a small image and get stretched otherwise. How do I do it in markdown?

Reviewer 2: Hmm I think the intention is for the image itself to be 300 pixels wide. @jorgeorpinel suggestions?

Reviewer: Minor, but I think @casperdcl is right, making the image that size is the best solution.

Reviewer 3: I think this is about retina-like display support. Unlike the blog engine, DVC.org doesn't process images properly. And effective you have to put extra white-background and do 2xWidth to them not look blurry and at the same moment be smaller then the viewport, or even of the same size but not blurry. E.g. https://dvc.org/doc/use-cases/versioning-data-and-model-files here images are wrong (not enough width to properly render them on the retina-like displays).

In blog we have a special logic and additional Markdown support for this (check the contributors guide).

@tapadipti 

- for now we can merge as is
- later (or if it's quick, you can either try and create images that are at least 1400px width (add transparent padding if needed)
- create a ticket (if don't have one) to get the image processing logic on docs to be on par with the blog's one cc @julieg18 @rogermparent

Reviewer: > > it's a small image and get stretched otherwise
>
> the intention is for the image itself to be resized to be 300 pixels wide rather

Actually, seeing https://dvc-org-studio-docs-vntpxfoqdz.herokuapp.com/doc/studio/visualize-experiments I'm not getting why we want the image to be so small.

These images are really small, hard to tell what's going on in there. It's already a bit of a problem in the **Create a View** page, but here they're quite unreadable:

![image](https://user-images.githubusercontent.com/1477535/117913649-6644ce80-b2a7-11eb-9617-6c400b34caa7.png)

Why not let them take the full width (already not a huge space) like in [/doc/studio/create-view#share-a-view](https://dvc-org-studio-docs-vntpxfoqdz.herokuapp.com/doc/studio/create-view#share-a-view)?

> /doc/use-cases/versioning-data-and-model-files here images are wrong (not enough width to properly render them on the retina-like displays

Idk about retina display (maybe a separate issue?) but the images in the Versioning use case look OK on my screen. They are 1026 px wide BTW, and only 14 KB (in case that's the concern here).

Reviewer 3: > but the images in the Versioning use case look OK on my screen

they are blurry on retina screens ... as I mentioned blog engine has a lot of specific mechanics to handle all of this properly. Including making them clickable (to read). Let's not address this for now here.

I agree though that it's fine to keep images bigger. Again, not a top priority. Let's focus on fixing bugs, something that we can quickly commit and move on. Everything else move to checkboxes.

Reviewer: > Let's focus on fixing bugs, something that we can quickly commit and move on.

TBH I haven't found anything that would prevent a merge here. Maybe for now we can keep some unresolved discussions and I'll move them to issues/checkboxes/PRs when this is merged?

> The values listed in this form by default are extracted from your selected
> commit. Change the input files or parameters according to what experiment you
> want to run. For instance, in the given example, you can increase `max_features`
> to 4000 or decrease `ngrams` to 1.

Reviewer 2: ```suggestion
The default values in this form are extracted from your selected commit. Change
the input files or parameters according to what experiment you want to run. For
instance, in the given example, you can increase `max_features` to 4000 or
decrease `ngrams` to 1.
```

> After making all the necessary input changes, click on Start commit. Enter your
> commit messages, select the branch you want to commit to (the same branch as the
> base branch or a new branch), and click on Commit changes.

Reviewer 2: ```suggestion
After making all the necessary input changes, click on `Start commit`. Enter
your commit messages, select the branch to commit to (either the base or a new
branch), and click on `Commit changes`.
```

> Now, if your refresh the View details page, the experiment (ie, the commit that
> you just pushed) will be included in the list of all commits, along with the
> results (metrics) of the experiment.

Reviewer 2: ```suggestion
If your refresh the `View details` page now, the experiment (i.e. the commit
that you just pushed) along with its results (metrics) will be included in the
list of all commits.
```
